src/Filtering/Smoothing/SmoothImageWithDiscreteGaussianFilter/Code.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+# Copyright NumFOCUS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itk
+import argparse
+
+parser = argparse.ArgumentParser(
+    description="Smoothing Image With Discrete Gaussian Filter."
+)
+parser.add_argument("input_image")
+parser.add_argument("output_image")
+parser.add_argument("sigma", type=float)
+args = parser.parse_args()
+
+PixelType = itk.UC
+Dimension = 2
+
+ImageType = itk.Image[PixelType, Dimension]
+
+reader = itk.ImageFileReader[ImageType].New()
+reader.SetFileName(args.input_image)
```

Comment:
Filters in this example could be replaced with pythonic equivalents -- `itk.imread` instead of `itk.ImageFilterReader[ImageType].New()`, `itk.discrete_gaussian_image_filter(reader.GetOutput(),sigma=args.sigma)`, and `itk.imwrite(image)`.